automation/update-from-application-services.py

```diff
@@ -58,10 +58,6 @@ def update_source(version):
         temp_dir = Path(temp_dir)
         extract_tarball(version, temp_dir)
         replace_all_files(temp_dir)
-    ## FIXME: https://github.com/mozilla/rust-components-swift/issues/106
-    print("Running format to format swift generated code")
-    subprocess.check_call(['swiftformat', 'swift-source/all/Generated'])
-    subprocess.check_call(['swiftformat', 'swift-source/focus/Generated'])
```

Comment:
I guess this means we should wait to merge this until the app-services code runs `swiftformat`.  I'll open a PR for that now.

Reply:
https://github.com/mozilla/application-services/pull/5565